lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -20,6 +20,25 @@ use List::Util qw(min);
 
 use constant DEPENDENCY_DEBUG_INFO => $ENV{OPENQA_DEPENDENCY_DEBUG_INFO};
 
+
+# inspired by assets/stylesheets/openqa_theme.scss
+# some colors changed to improve readability of white text
+my %BADGE_RESULT_COLORS = (
```

Comment:
Duplicating color definitions in scss and perl code looks like there should be a better way. What do others think?

Reply:
We agree. But we don't think there is. See https://github.com/os-autoinst/openQA/pull/5022#discussion_r1116904020